lighthouse-core/gather/gather-runner.js

```diff
@@ -107,16 +111,21 @@ class GatherRunner {
       .then(userAgent => {
         gathererResults.UserAgent = [userAgent];
         GatherRunner.warnOnHeadless(userAgent, gathererResults);
-        gathererResults.fetchedAt = [(new Date()).toJSON()];
       })
       .then(_ => driver.beginEmulation(options.settings))
       .then(_ => driver.enableRuntimeEvents())
       .then(_ => driver.cacheNatives())
       .then(_ => driver.registerPerformanceObserver())
       .then(_ => driver.dismissJavaScriptDialogs())
-      .then(_ => resetStorage && driver.clearDataForOrigin(options.url));
+      .then(_ => {
+        if (resetStorage) return driver.clearDataForOrigin(options.url);
```

Comment:
sorry, tsc rightly complains about the return type of `resetStorage && driver.clearDataForOrigin(options.url))`. Good news is this will be much nicer when we switch to async/await for these functions :)